README.md

```diff
@@ -33,7 +33,56 @@ this if you're using Chrome minus 2 or Node 7.6.0 or higher for `async`/`await`.
 
 ## Usage
 
-### Converting a callback to a promise
+## Converting a callback API to a promise API
+
+The `promisify` function is based off of Node 8's `util.promisify`, but has special support for
+callbacks with multiple values, and has utilities to create a copy of an object with
+promise-returning methods.
```

Comment:
Yep. The problem with mutating an API's object is that that API may have methods that rely on `this.otherMethod(..., callback)`